resources/etcd-member-dropin.conf

```diff
@@ -19,8 +21,6 @@ Environment="ETCD_PEER_CLIENT_CERT_AUTH=true"
 Environment="ETCD_PEER_TRUSTED_CA_FILE=/etc/etcd/ssl/ca.pem"
 Environment="ETCD_PEER_CERT_FILE=/etc/etcd/ssl/node.pem"
 Environment="ETCD_PEER_KEY_FILE=/etc/etcd/ssl/node-key.pem"
-Environment="RKT_RUN_ARGS=\
-  --uuid-file-save=${uuid_file} \
```

Comment:
As this line is going, you can remove the `uuid_file` argument from the template: 
https://github.com/utilitywarehouse/tf_kube_ignition/blob/46c85db4093a819ad2bb1253aa9cd79b31f9bf36/etcd.tf#L129

And the variable that switches between coreos and flatcar:
https://github.com/utilitywarehouse/tf_kube_ignition/blob/46c85db4093a819ad2bb1253aa9cd79b31f9bf36/variables.tf#L1-L4